speedy-recovery-client/src/components/core/App.js

```diff
@@ -17,10 +17,13 @@ class App extends Component {
       fhirClient: {},
       user: null,
       patient: {},
+      patients: [],
```

Comment:
This naming convention is a bit confusing. Maybe change patients to practitionersPatients or something similar?